sql/clients_last_seen_raw_v1.sql

```diff
@@ -0,0 +1,74 @@
+-- Equivalent to, but more efficient than, calling udf_bitmask_range(1, 28)
+CREATE TEMP FUNCTION bitmask_lowest_28() AS (0x0FFFFFFF);
+--
+CREATE TEMP FUNCTION shift_one_day(x INT64) AS (IFNULL((x << 1) & bitmask_lowest_28(), 0));
+--
+CREATE TEMP FUNCTION combine_days(prev INT64, curr INT64) AS (IFNULL(shift_one_day(prev), 0) + IFNULL(curr, 0));
```

Comment:
`shift_one_day` has an embedded `IFNULL`, making this one redundant
```suggestion
CREATE TEMP FUNCTION combine_days(prev INT64, curr INT64) AS (shift_one_day(prev) + IFNULL(curr, 0));
```